### DISCUSSION TITLE
how to disable hover effect in Avalonia?

### DISCUSSION THREAD

**Discussion by ghost** [AvaloniaUI/Avalonia#12823] [Q&A] [answered]
Hi, I'm having a problem how to turn off the feature when I move the mouse over any of the interactive Controls like Button, Textbox, ComboBoxItem, and ComboBox, it lights up or goes dark. to announce it's interactive, i want to turn it off because i want to make a separate one for it.

![image](https://github.com/AvaloniaUI/Avalonia/assets/85250803/74edb7e3-b054-4765-b765-27983c12893f)

I tried bringing it back to Border but it didn't work.

**Comment by timunie** [comment, marked as answer]
lookup the original styles in source and override the needed setters / DynamicResources

**Reply by ghost** [reply]
Thanks, I found it
